zellij-server/src/wasm_vm.rs

```diff
@@ -210,6 +210,8 @@ fn start_plugin(
         .unwrap()
         .map_dir("/data", plugin_own_data_dir.as_path())
         .unwrap()
+        .map_dir("/tmp", ZELLIJ_TMP_DIR.as_path())
+        .unwrap()
```

Comment:
@a-kenji Hi! While implementing the cool method you pointed out, I ran into a couple of situations.

In order for the plugin to access the local file system of the host machine, it must map the host directory to wasi_env. Therefore, in order to read the `VERSION` file, it seems to be necessary to map the user's `$HOME` directory.

This method is not bad. but if #894 is applied, it seems that the feature can be implemented very simply.

So, personally, I want to continue this work after #894 is merged. Coud I hear your opinion?

Reply:
Oh yeah, that is true. I think #894 looks good to me,
but I would like to have someone with a little more experience in the plugin system look over it as well.

I think it is a good idea, since that function would simplify the work here.